### ISSUE TITLE
Issues with Stability/URL Input

### ISSUE THREAD

**Issue by elrod-preston** [Mr-Bossman/CCminer-IOS#4]
On the latest release, the URL will not update when changed. Also, I found that starting and stopping the miner breaks the mining sometimes and the app has to be forced to close and open again. 

Background mining does not appear to be working but that could be because of a previously stated issue.

**Comment by Mr-Bossman** [comment]
>  Atarting and stopping the miner breaks the mining sometimes and the app has to be forced to close and open again.

Yes this is a known issue i tried fixing but still doesn't work sometimes

> Background mining does not appear to be working but that could be because of a previously stated issue.

I do not know why this is and i did know about it, i haven't had time to fix it yet.
